.github/workflows/notify-integration-release-via-tag.yaml

```
@@ -0,0 +1,54 @@
+name: Notify Integration Release (Tag)
+on:
+  push:
+    tags:
+      - '*.*.*'   # Proper releases
```

```suggestion
  push:
    tags:
      - 'v*.*.*'   # Proper releases
```
We only initiate releases when there is a leading `v` in the tag. We should probably follow the same practice here.

https://github.com/digitalocean/packer-plugin-digitalocean/blob/133cd6221b53e22de43c279e9b362336dc8d2546/.github/workflows/release.yml#L16

That also raises a question for me about timing. If both our release workflow and the notify release are triggered by pushing a tag, it seems like we may notify you of the release before it has actually completed. Are there any concerns there? Would using the `release` trigger make sense here?

https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#release

---

Prefixing with a `v` looks good to me, we could probably adopt this for our plugins as well, though the regex does catch it as-is, hence the step to remove the leading `v` from the tag (the doc servers reject requests for versions with a trailing `v`).

For timing, we also do it in this order, and you're correct that the docs will be signalled as needing update before the release finished. In practice we haven't experienced problems (we scrape the docs from the tag, so the commit already exists and we can fetch the data), but we could indeed end-up with a release advertised on the docs site, even if the actual release process failed for some reason. Running this after the release has completed is probably a better idea indeed, and something we should do as well, thanks for the heads-up.

---

Great. Thanks for looking into this. Let us know how the testing goes!